package/bin/splunk_ta_modinput_rh_perform_crd_operation.py

```diff
@@ -33,9 +33,9 @@ def read_file(self, confInfo):
         file_path = self.callerArgs.data['file_path'][0]
 
         if os.path.exists(file_path):
-            with open(file_path, 'r') as ckpt_file:
+            with open(file_path, 'rb') as ckpt_file:
                 file_content = ckpt_file.read()
-            confInfo['file_content'] = ('file_content', file_content)
+            confInfo['file_content'] = ('file_content', file_content.decode())
```

Comment:
This would be executed in Python2, right?  We would have to provide `.decode("utf-8")` because with Python2, the default ASCII would be used.